scripts/linters/test_files/valid.py

```diff
@@ -45,6 +45,7 @@
     rejecting_suggestion_for_invalid_content_ids_jobs)
 from core.jobs.batch_jobs import (                                   # pylint: disable=unused-import  # isort: skip
     remove_profile_picture_data_url_field_jobs)
+from core.jobs.batch_jobs import contributor_admin_stats_jobs        # pylint: disable=unused-import  # isort: skip
```

Comment:
Just curious, why do we need this change?

Reply:
If we don't do this changes,https://github.com/oppia/oppia/blob/develop/scripts/linters/python_linter_test.py#L67 this test fails. It requires for every job to be imported in this file.

Reply:
@kshitij01042002 Can you point to the documentation where this is mentioned?

Reply:
Actually I didn't find any documentation related to this but after pushing the changes [this](https://github.com/oppia/oppia/blob/develop/scripts/linters/python_linter_test.py#L67) test was failing. On debugging and analyzing the test, also taking reference of other beam job PRs. I figured the above mentioned behaviour.

Reply:
Hmm, that's weird. Can you add a comment in that file saying that every batch_jobs should be added to that file and give the reason for it also?